### DISCUSSION TITLE
ML Project With Open States Data

### DISCUSSION THREAD

**Discussion by mkilpat** [openstates/issues#651] [General] [answered]
Hi Open States, 
My partners and I are finishing a data science masters program at the Harvard Extension School and using the postgres data dump in our capstone project. We’re building a machine learning model to predict whether or not a bill will be passed into law based on the bill text and all the other bill data that’s scraped. We’re reaching out first to say thank you for making your data freely available! We are also curious to hear if any folks in the community have ideas about how ML could be used by Open States. Again, thank you and we’ll be sure to share our code and final results.
-Mike

**Comment by NewAgeAirbender** [comment, marked as answer]
Hey Mike,
Your capstone project sounds very interesting and we're happy we could provide your group with data! We'd definitely be interested in seeing your final results and discussing ideas about ML & OS.
Rylie
